tool/util/go.go

```
@@ -89,3 +99,12 @@ func SplitCompileCmds(input string) []string {
 func IsGoFile(path string) bool {
 	return strings.HasSuffix(strings.ToLower(path), ".go")
 }
+
+func NewFileScanner(file *os.File, size int) (*bufio.Scanner, error) {
+	if _, err := file.Seek(0, 0); err != nil {
+		return nil, ex.Wrapf(err, "failed to seek to beginning of build plan log")
```

"faild to seek file balah balah" instead of "build plan log"